requirements.in

```diff
@@ -6,8 +6,7 @@ paramiko>=2.10.1
 PyYAML
 ruamel.yaml
 git+https://github.com/containerbuildsystem/osbs-client@8d7d7fadff38c8367796e6ac0b3516b65483db24#egg=osbs-client
-# cannot build cryptography with rust for version >= 3.5
-cryptography < 3.5
+cryptography
```

Comment:
Wait, even cryptography? I wasn't aware of any workarounds for downloading the Rust dependencies

Reply:
Yeah, python-cryptography is now implemented in Rust => to build it from source, you need to be able to download the Rust dependencies. Which is not possible in OSBS, even with Cachito's help (so far) :disappointed: 

But maybe the cryptography project started vendoring the Rust deps or something, it's worth a shot

Reply:
I found this in docs, "On most Linux distributions, the latest version of pip will be able to install a binary wheel, so you won’t need a Rust toolchain." 

And if not, it might be possible to install rust packages as RPMs.

Reply:
Cachito only provides the source distributions, no pre-built binaries (otherwise we would be breaking legal/security requirements)

But installing from RPMs might work, in fact maybe you could just install python-cryptography from an RPM instead of getting it from Cachito. But you would have to make sure that the cryptography version pinned in requirements.txt is that same one that you can get from an RPM

Reply:
It seems like there isn't any good way to have newer cryptography version in osbs image. I kept the old one.